cse/lang/__init__.py

```diff
@@ -0,0 +1,95 @@
+"""
+CSE - a web crawling and web searching application for news paper
+comments written in Python
+
+simple language processing for transforming text into indexable tokens
+"""
+import abc
+
+from cse.lang.Preprocessor import Preprocessor
+from cse.lang.NltkTokenizer import NltkTokenizer
+from cse.lang.NltkStopwordFilter import NltkStopwordFilter
+from cse.lang.NltkStemmer import NltkStemmer
+from cse.lang.NltkLemmatizer import NltkLemmatizer
+
+
+class PreprocessorBuilder(object):
+
+    __useStopwords = False
+    __useStemming = False
+    __useLemmatizing = False
+
+    __stopwords = []
+    __tokenizer = None
+    __stemmer = None
+    __lemmatizer = None
+
+
+    def __init__(self):
+        self.__tokenizer = NltkTokenizer()
+
+
+    def useNltkTokenizer(self):
```

Comment:
It's clear that we need a tokenizer. But if a `useNltkTokenizer` method exists it should be optional (additionally it's already initialized in the `__init__` method)

Reply:
So would you suggest to silently use the `NltkTokenizer` as default (in the `__init__` method) if there was none specified? Or do you think, it's better to let the user decide and if none specified raise an exception (like in [line 72](https://github.com/CodeLionX/CommentSearchEngine/pull/19/files#diff-9e24383affb7ae83521fe57adb80f94fR72))?

Reply:
If we have a method `useTokenizer` we could define the NltkTokenizer as default. For now I would raise an exception.